incident/models/incident_index_page.py

```diff
@@ -228,7 +228,6 @@ def get_context(self, request, *args, **kwargs):
 
         context['entries_page'] = entries
         context['paginator'] = paginator
-        context['summary_table'] = incident_filter.get_summary()
```

Comment:
I think this can be removed from `CategoryPage` as well? Though doesn't affect the performance of the page much I guess.

Reply:
Yes, I think it can be removed, I'll do it